.github/workflows/whitespace.yml

```diff
@@ -0,0 +1,53 @@
+name: Whitespace
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  release:
```

Comment:
No idea if this makes sense. Do you?

Reply:
I copied this from other workflows, but don't know if we need the `release`...

Reply:
All right, maybe it's fine.